src/components/SimpleCard/markup.ts

```diff
@@ -0,0 +1,104 @@
+import type { ShopifyProduct, ShopifyVariant } from "./types"
+import type { SimpleCard } from "./SimpleCard"
+
+export function generateCardHTML(element: SimpleCard, product: ShopifyProduct) {
+  const hasDiscount =
+    element.discount &&
+    product.variants?.[0]?.compare_at_price &&
+    product.variants[0].compare_at_price > product.variants[0].price
+
+  return `
+    <div class="simple-card">
+      ${generateImageHTML(element, product)}
+      <div class="simple-card__content">
+        ${element.brand && product.vendor ? `<div class="simple-card__brand">${escapeHTML(product.vendor)}</div>` : ""}
+        <h3 class="simple-card__title">
+          <a href="/products/${product.handle}" class="simple-card__link">
```

Comment:
move anchor wrapping to be applied directly inside .simple-card and remove all other anchor usages in the card markup